command_line_rock_paper_scissors.py

```diff
@@ -66,4 +68,7 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    try:
+        main()
+    except exc.RockPaperScissorsError as e:
```

Comment:
Nice! A bit of extra fanciness since you're going this route:
```python
import sys
...

if __name__ == '__main__':
    try:
        main()
    except exc.RockPaperScissorsError as e:
        if '--debug' in sys.argv[1:]:
            raise e
        else:
            print(e)
```

This enables you to do this, where an uncaught `RockPaperScissorsError` would have its message printed (what it's doing right now):
`$ python command_line_rock_paper_scissors.py`
And this, where an uncaught `RockPaperScissorsError` will have the full traceback:
`$ python command_line_rock_paper_scissors.py --debug`